pkg/backporter/backporter_test.go

```diff
@@ -211,43 +212,55 @@ func TestCreateCloneHandler(t *testing.T) {
 	if err != nil {
 		t.Fatalf("error creating bug: %v", err)
 	}
-	clonedRelease := "4.4.z"
-	prunedReleaseSet := sets.NewString(allTargetVersions.List()...)
-	prunedReleaseSet.Delete(clonedRelease)
+	clonedRelease := "4.3.z"
+	intermediateRelease := "4.4.z"
+	prunedReleaseSetForSingleClone := sets.NewString(allTargetVersions.List()...)
+	prunedReleaseSetForSingleClone.Delete(intermediateRelease)
+	prunedReleaseSetForMultipleClones := sets.NewString(allTargetVersions.List()...)
+	prunedReleaseSetForMultipleClones.Delete(clonedRelease)
+	prunedReleaseSetForMultipleClones.Delete(intermediateRelease)
```

Comment:
You can simplify all of this by doing `sets.NewString(slice...).Delete(clonedRelease).Delete(intermetdiateRelease()`